examples/integrations/README.md

```diff
@@ -132,6 +147,12 @@ Integrations are created for an API using the `integration:create` command. The
 * **lifecycleEvents**: The lifecycle events that will trigger webhook. This is a list from the options: "API_SAVED", "API_PUBLISHED".
 * **additionalHeaders**: Custom HTTP headers to be sent with webhook notifications. Use the format "name: value" for each header.
 
+## Property: `region`
+AWS region where the API will be published. 
+* AWS regions: "us-east-1", "us-east-2", "us-west-1", "us-west-2", "eu-west-1", "eu-west-2", "eu-west-3", "eu-central-1", "eu-north-1",
+            "ap-east-1", "ap-south-1", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ap-northeast-2", "sa-east-1", "cn-north-1",
+            cn-northwest-1", "ca-central-1", "me-south-1"
+
```

Comment:
Please move the regions list to the description of the `region` property. There's currently no need for a separate section because only 1 integration in API/CLI uses this list.